### PR TITLE
Feat/file structure update

### PR DESCRIPTION
**Pull request by PReichetanz**

### CODE REVIEW

File: scripts/create.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import { getElement } from './utility.js'
+
+/* CREATE CARD FORM */
+export const createCardForm = getElement('createCardForm')
```
Comment (julialoeschel) [inline scripts/create.js:4]: do you also export const ? I thougt only functions, to use them in other files...

File: scripts/create.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          Answer:
+        </h2>
+        <p class="card__answer-text hidden mx-1rem" data-js="answer">
+          The answer is 42.
+        </p>
+        <ul class="card__tag-list">
+        </ul>
+        </div>`
+
+  return card
```
Comment (julialoeschel) [inline scripts/create.js:101]: Uh wow! nice!

File: scripts/navigation.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+function hideSection(section) {
+  section.classList.add('hidden')
+}
+
+function addHighlighting(icon) {
+  icon.classList.add('navigation-item--active')
+}
+
+function removeHighlighting(icon) {
+  icon.classList.remove('navigation-item--active')
```
Comment (julialoeschel) [inline scripts/navigation.js:89]: nice!!!!

**Review by julialoeschel — COMMENTED** [review]
really good work. 
I wonder why you export const in yout js, i never saw this before. everything else is super smooth :)

File: scripts/create.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+// TAKE INPUT VALUES OF FORM
+createCardForm.addEventListener('submit', () => {
+  const createCardQuestion = getElement('createCardQuestion')
+  const createCardAnswer = getElement('createCardAnswer')
+
+  const questionTextInput = createCardQuestion.value
+  const answerTextInput = createCardAnswer.value
+
+  // LOOP OVER NODELIST TO REACH HTML ELEMENTS ON NEW CARD
+  for (let i = 0; i < cardContainerChildren.length; i++) {
```
Comment (derjubu) [inline scripts/create.js:21]: WOW! Colour me impressed!

File: scripts/utility.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+export function getElement(dataJS, target = document) {
+  return target.querySelector(`[data-js=${dataJS}]`)
```
Comment (derjubu) [inline scripts/utility.js:2]: I see, you already adapted the information for the pages we read this weekend. Nice!

**Review by derjubu — APPROVED** [review]
Good job!

**Pull request merged**